src/upscaler.py

```diff
@@ -703,7 +706,16 @@ def run(self):
 
                 # downscale frames with Lanczos
                 Avalon.info(_('Lanczos downscaling frames'))
-                shutil.rmtree(self.extracted_frames)
+                try:
+                    shutil.rmtree(self.extracted_frames)
+                except Exception:
+                    if platform.system() == 'Windows':
+                        for file in self.extracted_frames.iterdir():
```

Comment:
Historically, `file` was a keyword, an alias for `open.
I would recommend using something more specific like `extracted_frame`.